x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/70_locale.yml

```diff
@@ -29,7 +29,7 @@ setup:
   - do:
       allowed_warnings_regex:
         - "No limit defined, adding default limit of \\[.*\\]"
-        - "Date format \\[MMMM] contains textual field specifiers that could change in JDK 23"
+        - "Date format \\[MMMM] contains textual field specifiers that could change in JDK 23.*"
```

Comment:
Two backslashes. Is that because this uses quotes instead of apostrophes?

Reply:
I believe so - it passes the tests either way

Reply:
Yeah, its `"` vs `'`